torch/onnx/symbolic_opset16.py

```diff
@@ -0,0 +1,41 @@
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
+# This file exports ONNX ops for opset 16
+
+# Note [ONNX Operators that are added/updated in opset 16]
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# https://github.com/onnx/onnx/blob/main/docs/Changelog.md#version-16-of-the-default-onnx-operator-set
+# New operators:
+#   GridSample https://github.com/onnx/onnx/pull/3557
+#
+# Updated operators:
+#    Identity
+#    If
+#    LeakyRelu
+#    Loop
+#    PRelu
+#    RoiAlign
+#    Scan
+#    ScatterElemenets
+#    ScatterND
+#    Where
+#    GreaterOrEqual
+#    LessOrEqual
+#    SequenceMap
+
+from torch.onnx.symbolic_helper import parse_args
+
+
+# note (mkozuki): `torch.nn.functional.grid_sample` calls `torch.grid_sampler`.
+@parse_args("v", "v", "s", "s", "b")
+def grid_sampler(g, input, grid, mode, padding_mode, align_corners):
+    return g.op(
+        "GridSample",
+        input,
+        grid,
+        align_corners_i=int(align_corners),
+        mode_s=mode,
```

Comment:
From test log, `test_grid_sampler` throws
```
onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] :
   6 : RUNTIME_EXCEPTION : Exception during initialization: /onnxruntime_src/onnxruntime/core/providers/cpu/tensor/grid_sample.h:21 
    onnxruntime::GridSample<T>::GridSample(const onnxruntime::OpKernelInfo&) [with T = float] 
      mode_str == "bilinear" || mode_str == "nearest" || mode_str == "bicubic" was false. 
      mode "tensor(0)" not supported, expect bilinear, nearest or bicubic
```

This is because PyTorch internally converts `mode` from string to enum flag, hence ORT is observing `tensor(0)` instead of a proper string. The map can be found here https://github.com/pytorch/pytorch/blob/3ac27e78ca5429b47a63826a1bb678031d20bffd/torch/nn/functional.py#L4008

Here I'd suggest based on their enum values, convert `mode` and `padding_mode` back to string that is compatible with ONNX spec https://github.com/onnx/onnx/blob/main/docs/Operators.md#gridsample